index.html

```diff
@@ -30,6 +30,44 @@ <h2>What I do</h2>
       <p>
         <a href="https://www.linkedin.com/in/olysu/">view full cv</a>
       </p>
+      <!--work section-->
+      <h2>work I've done</h2>
```

Comment:
Semantically, I think it'd be good to group components with the same content together with `section`.

With the way `section` is used at the moment, this new _work_ content is currently grouped in the same `section` as _what I do_.